.gitlab-ci.yml

```
@@ -96,6 +96,8 @@ cargo-check-benches:
   <<:                              *docker-env
   script:
     - BUILD_DUMMY_WASM_BINARY=1 time cargo check --benches --all
+    - cd srml/staking/
+    - time cargo test --release --verbose --no-default-features --features bench
```

More like this please :p 

```suggestion
    - time  cargo bench --features bench --verbose
```

Staking benches just need the bench feature, no need to kill std.